swan-lake/resources/learn-the-language/language-basics.md

```diff
@@ -245,7 +245,7 @@ int n = string:length(s);
 
 ## Arrays
 
-Arrays are sequential data structures consisting of values of the same type.  
+Arrays are ordered data structures consisting of values of the same type.  
```

Comment:
We need to include `optionally with a specific length` part also in here.
Tracked in https://github.com/ballerina-platform/ballerina-dev-website/issues/8676